be/src/storage/tablet_updates.cpp

```diff
@@ -1804,8 +1805,226 @@ Status TabletUpdates::load_from_base_tablet(int64_t request_version, Tablet* bas
     index.unload();
     update_manager->index_cache().release(index_entry);
     _tablet.set_tablet_state(TabletState::TABLET_RUNNING);
-    LOG(INFO) << "load_from_base_tablet finish tablet:" << _tablet.tablet_id() << " version:" << this->max_version()
+    LOG(INFO) << "schema_change_linked finish tablet:" << _tablet.tablet_id() << " version:" << this->max_version()
+              << " #pending:" << _pending_commits.size();
+    return Status::OK();
+}
+
+Status TabletUpdates::schema_change_directly(int64_t request_version, const std::shared_ptr<Tablet>& base_tablet,
+                                             vectorized::ChunkChanger* chunk_changer) {
+    DCHECK(_tablet.tablet_state() == TABLET_NOTREADY)
+            << "schema_change_directly is only allowed in schema change process";
```

Comment:
tablet state is NOTREADY, schema change is not allowed. + tablet_id

Reply:
the tablet state must be TABLET_NOTREADY in schema change process, the DCHECK output indicate that THIS function can not be a callee in non-schema change process, two proposals:
```
DCHECK(_tablet.tablet_state() == TABLET_NOTREADY)
            << "link_from is only allowed in schema change process(tablet state is not TABLET_NOTREADY)"
            << ", tablet_id: " << _tablet.tablet_id() << ", tablet state: " << _tablet.tablet_state();
```
or
```
DCHECK(_tablet.tablet_state() == TABLET_NOTREADY)
            << "link_from is only allowed in schema change process"
            << ", tablet_id: " << _tablet.tablet_id() << ", tablet state: " << _tablet.tablet_state();
```